os/root-on-raid.md

```diff
@@ -0,0 +1,58 @@
+# Setting up Root on Raid
+
+Partitions used in RAID arrays need to have the gpt partition type uuid `be9067b9-ea49-4f15-b4f6-f36f8c9e1818` in order to boot. Because Ignition does not current support modifying existing partitions
+without wiping the partition table, the default ROOT partition cannot be used since it has different partition type uuid. In order to have the root partition in a raid array:
+ * Create the partitions used in the RAID array with the gpt partition type uuid `be9067b9-ea49-4f15-b4f6-f36f8c9e1818`
+ * Create a RAID array with the created partitions 
+ * Create a filesystem with on the raid array with the ROOT label
+ * Remove the ROOT label from the old root filesystem by wiping the existing filesystem or overwriting it
+
+## Example CLC
+
+```yaml container-linux-config
+storage:
+  disks:
+    - device: /dev/vdb
+      wipe_table: true
+      partitions:
+       - label: root1
+         number: 1
+         size: 256MiB
+         start: 0
+         type_guid: be9067b9-ea49-4f15-b4f6-f36f8c9e1818
+       - label: root2
+         number: 2
+         size: 256MiB
+         start: 0
+         type_guid: be9067b9-ea49-4f15-b4f6-f36f8c9e1818
+  raid:
+    - name: "my_raid_array"
+      level: "raid1"
+      devices:
+        - "/dev/vdb1"
+        - "/dev/vdb2"
```

Comment:
Let's not recommend, even implicitly, that the user RAID together two partitions on the same disk.